.github/renovate.extension.json

```diff
@@ -0,0 +1,36 @@
+{
+    "extends": [
+      "config:recommended",
+      ":semanticCommitTypeAll(chore)",
+      "helpers:pinGitHubActionDigests"
+    ],
+    "commitMessagePrefix": "deps: ",
+    "ignorePresets": [":semanticPrefixFixDepsChoreOthers"],
+    "ignorePaths": ["docs/", "data/"],
+    "prConcurrentLimit": 0,
+    "rebaseWhen": "behind-base-branch",
+    "dependencyDashboard": true,
+    "dependencyDashboardLabels": ["type: process"],
+    "semanticCommits": "disabled",
+    "pip_requirements": {
+      "fileMatch": ["extension_service/requirements.txt", "extension_service/requirements-test.txt"]
```

Comment:
Shouldn't this cover the github repo as well?

also super-nit: prefer to use a multi line list here to make it easier to add/remove in the future